editor/src/core/shared/element-template.ts

```diff
@@ -1066,6 +1066,14 @@ export function clearJSXFragmentWithoutUIDUniqueIDs(
 
 export type ElementsWithin = { [uid: string]: JSXElement }
 
+export interface WithElementsWithin {
```

Comment:
Could have the types that have this extend this interface.

Reply:
aha! yeah good idea https://github.com/concrete-utopia/utopia/pull/3783/commits/d6f69f0101b8e160720fc2f9d2216496d6d32911